{{cookiecutter.project_slug}}/home/api/v1/serializers.py

```diff
@@ -37,6 +37,10 @@ def create(self, validated_data):
         user.save()
         return user
 
+    def save(self, request=None):
```

Comment:
Rest auth register serializer does bunch of stuff in save method to link it to allauth flow. I think we should do that.

Reply:
I looked through the calls and the only one that does anything is `setup_user_email` which I've added.